### ISSUE TITLE
Cannot run demo locally

### ISSUE THREAD

**Issue by scriptdaemon** [ng-bootstrap/ng-bootstrap#1045]
### Bug description:

Using the instructions here https://github.com/ng-bootstrap/ng-bootstrap/blob/master/DEVELOPER.md

After running `npm install`, when trying to run `$(npm bin)/gulp demo-server` I get the following errors (which display in the command prompt and in the browser when loading http://localhost:9090/):

```
$ $(npm bin)/gulp demo-server
[10:05:40] Using gulpfile ~\Documents\Repository\ng-bootstrap\gulpfile.js
[10:05:40] Starting 'generate-docs'...
[10:05:41] Starting 'generate-plunks'...
[10:05:41] Finished 'generate-docs' after 1.07 s
[10:05:41] Finished 'generate-plunks' after 306 ms                              10% building modules 3/3 modules 0 active                                 chunk    {0} js/vendor.js (vendor) 40 bytes {2} [initial] [rendered]                  chunk    {1} js/app.js (app) 40 bytes {0} [initial] [rendered]                  chunk    {2} js/polyfills.js (polyfills) 227 kB [entry] [rendered]              
ERROR in multi polyfills
Module not found: Error: Can't resolve 'ts' in 'C:\Users\kwilliams\Documents\Repository\ng-bootstrap'
 @ multi polyfills

ERROR in multi vendor
Module not found: Error: Can't resolve 'ts' in 'C:\Users\kwilliams\Documents\Repository\ng-bootstrap'
 @ multi vendor

ERROR in multi app
Module not found: Error: Can't resolve 'ts' in 'C:\Users\kwilliams\Documents\Repository\ng-bootstrap'
 @ multi app

ERROR in   Error: Child compilation failed:
  Entry module not found: Error: Can't resolve 'raw' in 'C:\Users\kwilliams\Docu  ments\Repository\ng-bootstrap':
  Error: Can't resolve 'raw' in 'C:\Users\kwilliams\Documents\Repository\ng-boot  strap'

  - compiler.js:76
    [ng-bootstrap]/[html-webpack-plugin]/lib/compiler.js:76:16

  - Compiler.js:282 Compiler.<anonymous>
    [ng-bootstrap]/[webpack]/lib/Compiler.js:282:10

  - Compiler.js:479
    [ng-bootstrap]/[webpack]/lib/Compiler.js:479:13

  - Tapable.js:81 next
    [ng-bootstrap]/[webpack]/[tapable]/lib/Tapable.js:81:11

  - CachePlugin.js:61 Compiler.<anonymous>
    [ng-bootstrap]/[webpack]/lib/CachePlugin.js:61:4

  - Tapable.js:85 Compiler.applyPluginsAsync
    [ng-bootstrap]/[webpack]/[tapable]/lib/Tapable.js:85:13

  - Compiler.js:476
    [ng-bootstrap]/[webpack]/lib/Compiler.js:476:10

  - Tapable.js:73 Compilation.applyPluginsAsync
    [ng-bootstrap]/[webpack]/[tapable]/lib/Tapable.js:73:70

  - Compilation.js:627
    [ng-bootstrap]/[webpack]/lib/Compilation.js:627:18

  - Tapable.js:73 Compilation.applyPluginsAsync
    [ng-bootstrap]/[webpack]/[tapable]/lib/Tapable.js:73:70

  - Compilation.js:618
    [ng-bootstrap]/[webpack]/lib/Compilation.js:618:10

  - Tapable.js:73 Compilation.applyPluginsAsync
    [ng-bootstrap]/[webpack]/[tapable]/lib/Tapable.js:73:70

  - Compilation.js:613
    [ng-bootstrap]/[webpack]/lib/Compilation.js:613:9

  - Tapable.js:73 Compilation.applyPluginsAsync
    [ng-bootstrap]/[webpack]/[tapable]/lib/Tapable.js:73:70

  - Compilation.js:609
    [ng-bootstrap]/[webpack]/lib/Compilation.js:609:8

  - Tapable.js:73 Compilation.applyPluginsAsync
    [ng-bootstrap]/[webpack]/[tapable]/lib/Tapable.js:73:70

  - Compilation.js:555 Compilation.seal
    [ng-bootstrap]/[webpack]/lib/Compilation.js:555:7

  - Compiler.js:473
    [ng-bootstrap]/[webpack]/lib/Compiler.js:473:16

  - Tapable.js:152
    [ng-bootstrap]/[webpack]/[tapable]/lib/Tapable.js:152:11

  - Compilation.js:452 Compilation.<anonymous>
    [ng-bootstrap]/[webpack]/lib/Compilation.js:452:10

  - Compilation.js:347 Compilation.errorAndCallback
    [ng-bootstrap]/[webpack]/lib/Compilation.js:347:3

  - Compilation.js:364 Compilation.<anonymous>
    [ng-bootstrap]/[webpack]/lib/Compilation.js:364:11


Child html-webpack-plugin for "index.html":

    ERROR in Entry module not found: Error: Can't resolve 'raw' in 'C:\Users\kwilliams\Documents\Repository\ng-bootstrap'
webpack: bundle is now VALID.
```

### Link to minimally-working plunker that reproduces the issue:

N/A

### Version of Angular, ng-bootstrap, and Bootstrap:

Angular:

@angular/compiler: 2.0.0
@angular/compiler-cli: 0.6.2
@angular/http: 2.0.0
@angular/platform-browser: 2.0.0
@angular/platform-browser-dynamic: 2.0.0
@angular/platform-server: 2.0.0
@angular/router: 3.0.0

ng-bootstrap:

1.0.0-alpha.13

Bootstrap:

^4.0.0-alpha.5

**Comment by pkozlowski-opensource** [comment]
Are you sure that you are using the suggested node / npm version: https://github.com/ng-bootstrap/ng-bootstrap/blob/master/DEVELOPER.md#prerequisite-software ?

In generally I'm not sure we can help troubleshooting every existing dev environment...  Our take here is that people wanting to change demo locally are actual library developers and as such should be able to _help us_ improve build to suit their environment.

In short: we would gladly merge PRs that fix problems with your particular environment if our build system is faulty. But we can't debug all the possible dev environments for a very simple reason - we don't have access to your environment so we can't debug / troubleshoot it. In other words you are in the best position to figure out and fix the problem (!).

Closing per above explanation.
